### ISSUE TITLE
Comments and #defines prevent extension from working

### ISSUE THREAD

**Issue by pfalson** [mike-lischke/vscode-antlr4#108]
If you have a grammar with

/*
comment
*/

The grammar isn't recognised as a valid grammar. 

If you don't have visualParseTree = true then it just runs with no output (giving no clue what's going on).

Additionally (and not sure this is a bug per se as #define isn't really grammar):

#define mydef 1

also stops it working.

**Comment by mike-lischke** [comment]
Almost in every grammar I work in there's at least one comment with the `/* */` style. That means the comment as such isn't the deal break. Can you a post a minimal grammar that shows the behavior you describe here?

The `#define` is clearly a syntax error and I wonder you write that in the grammar in the first place.

And what do you mean by "stops it working"? What exactly does not work anymore (syntax highlighting, debugging, code completion etc.)?

**Comment by pfalson** [comment]
Hmmm...so weirdly I reverted all the changes I made to my grammar and now your tool is working. I'm thinking this might be something in vscode.

_So....maybe just close this._

When I mentioned stops working the run/debug just goes through to completion and there's no output in the debug console.

**Comment by pfalson** [comment]
@mike-lischke Just an FYI, I do continue to have issues running the tool (it IS a great tool!!).

There seems to be a change/ready state issue. My previous report (above) was erroneous. The #define and comments are not the issue. The reason it worked after I removed them was that I had made a change to the file.

So every now and then if the debug/run doesn't work I make a comment/white space change which seems to wake it up.
